### ISSUE TITLE
(zhixianyan): no need to limit output_arrays to `Softmax' 

### ISSUE THREAD

**Issue by piascikj** [imdone/tensorflow#2017]
once b/74205001 fixed and argmax implemented in tflite.

----
*opened via [imdone.io](https://imdone.io) from a code comment on [db076ca01f12368c9476fa4db9d87756f22f9670](https://github.com/imdone/tensorflow/commit/db076ca01f12368c9476fa4db9d87756f22f9670) by Suharsh Sivakumar suharshs@google.com*

----

https://github.com/imdone/tensorflow/blob/8cb6e535f6ea14b380aa86d425169adad682cb8c/tensorflow/contrib/lite/python/convert_saved_model_test.py#L383-L389
